### ISSUE TITLE
Why is this printing `undefined`?

### ISSUE THREAD

**Issue by Zearin** [docpad/docpad#952]
I’ve tried to track down why there is a line that says `undefined` in the following, but without success.

```
info: Welcome to DocPad v6.78.1 (local installation: /Users/«…»/node_modules/docpad)
notice: If everyone who saw this message donated $1/week, maintaining DocPad would become sustainable: http://docpad.org/donate
info: Contribute: http://docpad.org/docs/contribute
info: Plugins: coffeescript, eco, ect, growl, livereload, marked, stylus
info: Environment: development
info: DocPad listening to http://127.0.0.1:9778/ on directory /Users/«…»/out
info: LiveReload listening to new socket on channel /docpad-livereload
undefined
info: Generating...
info: Generated 50/51 files in 27.91 seconds
info: Watching setup starting...
info: Watching setup
info: The action completed successfully
```

Anyone know what’s happening?  

If not, how do I track this down?  

(I’ve tried using `--debug`, without success. At varying times I’ve gotten: too much output, a failure to find anything containing `undefined` in the output, or errors raised from the use of `--debug` itself.)

**Comment by mikeumus** [comment]
My current DocPad project is doing this too but I haven't bothered exploring it yet :confused:.

**Comment by balupton** [comment]
I'm not seeing this. I guess fixed in 6.80
